shared/health/src/main/java/io/pravega/shared/health/ContributorNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
```

Comment:
Please fix your header formatting in all new files.

Reply:
Yup, just realized now. It's was a default setting in IntelliJ that overwrote my file template. Fixing.